example/src/Examples/Animation/SpringBackTouch.tsx

```diff
@@ -48,6 +48,9 @@ export const SpringBackTouchAnimation = () => {
       rectY.current = y - offsetY.current;
     },
     onEnd: ({ velocityX, velocityY }) => {
+      if (velocityX == null || velocityY == null) {
+        return;
```

Comment:
It might be better to find out why the touch handler passes null for these values - maybe that could help others as well.

Reply:
This might be solved by looking in the file `useTouchHandler.ts` on lines 59..60?

Reply:
And thanks so much for submitting a PR, @gaishimo !! We really appreciate it!